docker-compose.yml

```diff
@@ -44,8 +44,11 @@ services:
   android-maze-runner:
     image: 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner-releases:latest-v3-cli
     environment:
-      VERBOSE:
       DEBUG:
+      BUILDKITE:
```

Comment:
Means that MazeRunner will be able to detect it being run from BK and use ANSI logging for the BrowserStack dashboard link.